packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -19,7 +19,7 @@ import { ENTER, BACKSPACE, DELETE } from '@wordpress/keycodes';
 import { __, sprintf } from '@wordpress/i18n';
 import { useSelect, useDispatch } from '@wordpress/data';
 import deprecated from '@wordpress/deprecated';
-import { getBlockProps } from '@wordpress/blocks';
+import { __unstabelGetBlockProps as getBlockProps } from '@wordpress/blocks';
```

Comment:
typo :) I think without the typo, the docs wouldn't have been included in the README :)

Reply:
Oops :)